### PR TITLE
add i18n and implement it in LoginPage.jsx

### PR DESCRIPTION
**Pull request by aleneagu96**

### CODE REVIEW

File: src/views/login/LoginPage.jsx [hunk 1]
```diff
@@ -20,6 +21,8 @@ const useStyles = makeStyles({
 });
 
 function LoginPage({ setAuthenticated }) {
+
+  const { t, i18n } = useTranslation();
```
Comment (andregeorgi) [inline src/views/login/LoginPage.jsx:25]: The i18n constant can be deleted since we are not using it anywhere.

File: src/views/todo/Todo.jsx [hunk 2]
```diff
@@ -40,8 +42,10 @@ const useStyles = makeStyles({
   },
 });
 
+
 const Todo = () => {
   const classes = useStyles();
+  const { t, i18n } = useTranslation();
```
Comment (andregeorgi) [inline src/views/todo/Todo.jsx:48]: Also here

**Review by andregeorgi — APPROVED** [review]

File: src/views/login/LoginPage.jsx (hunk 1 above)
Comment (sorincretescu) [reply src/views/login/LoginPage.jsx:25]: How about line 71 and 72?

File: src/views/todo/Todo.jsx (hunk 2 above)
Comment (sorincretescu) [reply src/views/todo/Todo.jsx:48]: Lines 208, 209

**Review by sorincretescu — APPROVED** [review]
Extraordinary work. One piece of advice tho': it is better to have a different file with the object that contains translations. You could just import it in the resources and have it as "en: englishTranslations" etc. Easier to read and it is separation of static data and code, which is a good practice. Also, if you want to translate "Add Todo", it is better to use t("Add Todo") then t("addTodo"). Looks better, easier to use, makes more sense. If you did this worrying about the translation object, keep in mind that you can always use quotes and have your object as:
{
    "Add Todo": "添加任務",
    "Whatevs":  "我不敢相信你為此浪費了時間",
}

Ok, you got me, these were more pieces of advice. Awesome work, tho'! ^^

**Pull request merged**